phpunit/block-bindings-test.php

```diff
@@ -310,7 +310,7 @@ public function test_update_block_with_value_from_source_image_placeholder() {
 			"The 'caption' attribute should be updated with the value returned by the source."
 		);
 		$this->assertSame(
-			'<figure class="wp-block-image"><img src="https://example.com/image.jpg" alt=""/><figcaption class="wp-element-caption">Example Image</figcaption></figure>',
+			'<figure class="wp-block-image"><img src="https://example.com/image.jpg" alt=""><figcaption class="wp-element-caption">Example Image</figcaption></figure>',
```

Comment:
This will be interesting to backport to `wordpress-develop` 🤔 

The corresponding file is https://github.com/WordPress/wordpress-develop/blob/70d00508e950f0e63de82386d453bca5ad899474/tests/phpunit/tests/block-bindings/render.php. We'll probably need to untangle it: Right now, it's not really a unit test for Block Bindings but more of an integration test that also covers the behavior of individual blocks (among them the Image block).

Test coverage for blocks should live in the GB repo -- as the implementation of blocks also does -- rather than in `wordpress-develop`.